core/model/survey.go

```diff
@@ -18,6 +18,23 @@ import (
 	"time"
 )
 
+// SurveyAlert is a survey alert to be sent to notifications BB
+type SurveyAlert struct {
+	ContactKey string                 `json:"contact_key" bson:"contact_key"`
+	Content    map[string]interface{} `json:"content" bson:"content"`
+}
+
+// AlertContact is what will be used to identify where to send survey alerts
+type AlertContact struct {
+	ID          string                 `json:"id" bson:"_id"`
+	Key         string                 `json:"key" bson:"key"`
+	Type        string                 `json:"type" bson:"type"`
+	Identifier  string                 `json:"identifier" bson:"identifier"`
```

Comment:
Let's rename `Identifier` to `Address` to make it clearer what this is supposed to be. We should also add `AppID` and `OrgID` fields and set/filter by them using the claims of the admin. Sorry for the change from the original spec!